src/test/ensemble_tests.cpp

```diff
@@ -2195,3 +2196,322 @@ TEST_F(EnsembleFlowTest, WaitForLoadingPipelineDefinitionFromBeginStatus) {
     t.join();
     t2.join();
 }
+
+const char* configJsonWithNoPipeline = R"(
```

Comment:
In my MR, I have created similar variable for other unit test and encountered symbol duplication compile time error. How about using `static const char*` to keep this declaration within one compilation unit? It resolved problem for me.